src/Service/RequestService.php

```diff
@@ -702,6 +712,10 @@ public function proxyHandler(array $data, array $configuration, ?Source $proxy =
             $this->data['path'] = '';
         }
 
+        if (count($data['endpoint']->getFederationProxies()) > 1) {
```

Comment:
Why specificly 1 and not 0?

Reply:
This is future proofing, in the end I would like to merge the federationProxies and proxy-properties of the Source entity, and when that happens, I want this to be the point on which we decide if we are doing a single source request, or a federated request.